docs/data-guidelines.md

```diff
@@ -172,6 +174,30 @@ For Safari for iOS, use the iOS version number, not the Safari version number or
 
 This versioning scheme came at [Apple's request, in #2006](https://github.com/mdn/browser-compat-data/issues/2006#issuecomment-457277312).
 
+## Addition of browsers
+
+BCD's [owners](https://github.com/mdn/browser-compat-data/blob/master/GOVERNANCE.md) may choose to adopt a new browser or engine. To add a new browser to BCD, we need evidence of (in decreasing order of importance):
+
+- a compelling downstream-consumer story (e.g., MDN or caniuse express an interest, or someone is planning to do something with the data that might plausibly grow BCD's reach)
+- reviewers (e.g., two or more people with interest and ability to test data relating to new and existing releases, or at least one reviewer acting on behalf of the vendor)
+- published release information (e.g., release notes with version numbers and dates)
```

Comment:
> Yeah, I was trying to capture something like this with the published release information proposal. I wanted something a bit more substantial than being able to populate a browser data file. I want to know about release process and plans too. I'm open to rewording this, if you've got any ideas.

Here's an attempt. Please let me know if we can make this even better.

```suggestion
- a release process allowing BCD to publish release information in a `browsers/` file (containing release notes with version numbers and dates. Merely "nightly" builds and/or experimental releases aren't satisfactory.)
```

Reply:
The _merely_ and "nightly" stuff reads as a little hostile to me. Maybe this gets the point across?

```suggestion
- a release process allowing BCD to publish stable release information in a `browsers/` file (containing release notes with version numbers and dates)
```

Reply:
> The _merely_ and "nightly" stuff reads as a little hostile to me. Maybe this gets the point across?

Yes. And with that, @Elchi3 and @ddbeck, I hope you won’t mind me taking a moment to interject a comment saying how genuinely glad I am for everything that the two of you bring to this effort that we’re all investing time together to make really great. In any really useful and really successful project, there’s a lot that ends up getting taken for granted, and I think we often have a tendency not to realize that the way the magic really happens is through some really smart, thorough, thoughtful leadership — day in and day out. So, thanks for everything you do in guiding this project in a way that always leaves me feeling like things are in really good hands, and I’m spending my time in the right place.

Reply:
Thanks so much @sideshowbarker! :heart: 
I'm very grateful for the chance to work with you, Daniel and everyone in the BCD and MDN community.

Reply:
Thank you for this kind comment, @sideshowbarker! Glad to be here too.